README.md

```diff
@@ -3029,6 +3029,15 @@ The `apache::vhost` defined type uses `concat::fragment` to build the configurat
 
 For the custom fragment's `order` parameter, the `apache::vhost` defined type uses multiples of 10, so any `order` that isn't a multiple of 10 should work.
 
+> **Note:** When creating an `apache::vhost`, it cannot be named `default` or `default-ssl`, because vhosts with these titles are always managed by the module. This means that you cannot override the `default` or `default-ssl` vhosts. An optional workaround is to create a vhost named something else, such as `my default`, and ensure that the `default` and `default_ssl` vhosts are set to `false`:
```

Comment:
I would probably take out "This means that you cannot override the `default` or `default-ssl` vhosts" as this is precisely what `default_vhost => false` is for. Maybe it should be "... `Apache::Vhost['default']` or `Apache::Vhost['default-ssl']` resources" to be more specific about what cannot be overridden?